internal/server/logic/events.go

```diff
@@ -44,7 +45,7 @@ func Analyze(eventKey string, teams []string, schema analysis.Schema, rs report.
 		}
 
 		if len(stats) == 0 {
-			return []TeamAnalysis{}, nil
```

Comment:
Why?

Reply:
Previously if there wasn't stats available for one of the teams on the alliance it just didn't return any averages for the whole alliance. Now it skips over teams with no data available, but includes stats for the other teams on the alliance.

Reply:
:+1: